Header header
sensor_msgs/Image image
int8 local_counter
float64[] centers
